src/homepageExperience/components/steps/go/WriteData.tsx

```diff
@@ -48,7 +48,8 @@ export const WriteDataComponent = (props: OwnProps) => {
     onSelectBucket(bucket.name)
   }, [bucket, onSelectBucket])
 
-  const codeSnippet = `org := "${org.name}"
+  const codeSnippet = `// FOO
```

Comment:
I see the `//  FOO` line is the only thing changed here, is that for consistency?

Reply:
This was removed